lisa/tools/ssh.py

```diff
@@ -40,3 +43,21 @@ def enable_public_key(self, public_key: str) -> None:
             self.node.get_pure_path("~/.ssh/authorized_keys"),
             append=True,
         )
+
+    def get_path_sshd_config_file(self, file_name: str = "sshd_config") -> str:
```

Comment:
`get_sshd_config_path`? The parameter is not necessary, the file name is decided by sshd service, it cannot be changed. Don't over design.

Reply:
updated